### PR TITLE
[HAC-2113] LabelList utility component

### PR DESCRIPTION
**Pull request by vidyanambiar**
Jira issue: [HAC-2113](https://issues.redhat.com/browse/HAC-2113)

This PR is for providing a LabelList utility that can be used to construct the details page for a resource.

Test with `yarn storybook`

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (florkbr) [inline .vscode/launch.json:2]: @vidyanambiar did you intend to check in this vscode config?

Comment (vidyanambiar) [reply .vscode/launch.json:2]: @florkbr Yes - I thought it would make it easy to debug in the future. If it isn't helpful, I can pull it out. LMK

**Pull request merged**